main.fmf

```diff
@@ -0,0 +1,2495 @@
+/check-accounts:
```

Comment:
> I like the general direction, this just needs some fixes to the implementation. I only realized at the fourth commit ("rework structure") that the commits are not really independent, and that I should probably have looked at the entire diff at once. So my comments will probably be shown as obsolete, but they are still relevant. Can you please squash this, split out the independent commits, and drop the generated files? Then this will be much easier to review in the second round.
> 
> Thanks!

Yes, I did not squashed commits, because I wanted to there way to be able to go back in the history in case some new solution will be worser, than new one :-)
and yes, this rework totally changed logic and structure and also your point why FMF and pytest wrapper are together.